docs/sphinx/source/whatsnew/v0.11.0.rst

```diff
@@ -23,7 +23,9 @@ Enhancements
   shade perpendicular to ``axis_azimuth``. The function is applicable to both
   fixed-tilt and one-axis tracking systems.
   (:issue:`1689`, :pull:`1725`, :pull:`1962`)
-
+* Add function :py:func:`pvlib.albedo.albedo_water`, to calculate the
```

Comment:
I find the name a bit repetitive. If it's going to end in the `pvlib.albedo` namespace, I suggest renaming the function to `inland_water`. `albedo_water` does not exclude the ocean, and that may be a nice clarification.

Reply:
I think even more specificity is needed.  This inland water albedo model is one of many that pvlib could potentially implement, so I think we should leave room for them.  Assuming the model itself doesn't have a name (I did not see one in the reference or in any PV papers citing it), how about `inland_water_dvoracek`?  I don't like it, but at least it is specific...